TWIRF4 Workshops (#71)
RustFestEU/blog.rustfest.eu: _posts/2018-11-02-talks-and-workshops.md

---
layout: post

> title: "This Week in RustFest 4: Workshops and Talk Schedule"

Reviewer: ```suggestion
title: "This Week in RustFest 4: Workshops and Talks Schedule"
```

socialImageSrc: "/assets/posts/rome/twirf-4.png"
authors:
  - gandro
---

## Last Batch of Tickets

We have opened our third and last batch of tickets. So get them while they are still available - you’ll be able to purchase your tickets via [🎟 this link 🎟](https://ti.to/asquera-event-ug/rustfest-rome-2018/).

# Talks and Workshops at RustFest Rome

> Last week, we finally announced the long awaited talk lineup for Rome on our Twitter accunt [@RustFest](https://twitter.com/RustFest) account. We also published a sneak peak of the schedule, which is now complete and available on our website:

Reviewer: ```suggestion
Last week, we finally announced the long awaited talk lineup for Rome on our Twitter account [@RustFest](https://twitter.com/RustFest) account. We also published a sneak peak of the schedule, which is now complete and available on our website:
```

> Last week, we finally announced the long awaited talk lineup for Rome on our Twitter account [@RustFest](https://twitter.com/RustFest) account. We also published a sneak peak of the schedule, which is now complete and available on our website:

Reviewer: ```suggestion
Last week, we finally announced the long awaited talks lineup for Rome on our Twitter account [@RustFest](https://twitter.com/RustFest) account. We also published a sneak peek of the schedule, which is now complete and available on our website:
```

<div style="text-align: center; padding: 1em;">
  <a href="https://rome.rustfest.eu/schedule/">
    <strong>📅🎙️📅 Talk Schedule 📅🎙️📅</strong>
  </a>
</div>

> This and next week, we are also releasing the workshop program for Sunday. We are going to have a wide range of workshop topics again this time, both for Rust beginner and experts alike. Follow us on Twitter and watch this spot for future updates regarding workshops:

Reviewer: ```suggestion
This and next week, we are also releasing the workshop program for Sunday. We are going to have a wide range of workshop topics again this time, both for Rust beginners and experts alike. Follow us on Twitter and watch this spot for future updates regarding workshops:
```

> This and next week, we are also releasing the workshop program for Sunday. We are going to have a wide range of workshop topics again this time, both for Rust beginners and experts alike. Follow us on Twitter and watch this spot for future updates regarding workshops:

Reviewer: ```suggestion
This and next week, we are also releasing the workshops program for Sunday. We are going to have a wide range of workshop topics again this time, both for Rust beginners and experts alike. Follow us on Twitter and watch this spot for future updates regarding workshops:
```
